CMakeLists.txt

```diff
@@ -58,6 +57,7 @@ else()
 		/usr/include/libevdev-1.0)
 endif()
 
+set(Boost_USE_STATIC_LIBS ON)
```

Comment:
This seems to be needed on my MinGW Windows build too in order to avoid the same linker errors we encountered on Ubuntu.